streams/readable-streams/owning-type-message-port.any.js

```diff
@@ -0,0 +1,51 @@
+// META: global=window,worker
+// META: script=../resources/test-utils.js
+// META: script=../resources/rs-utils.js
+'use strict';
+
+promise_test(async () => {
+  const channel = new MessageChannel;
+  const port1 = channel.port1;
+  const port2 = channel.port2;
+
+  const source = {
+    start(controller) {
+      controller.enqueue(port1, { transfer : [ port1 ] });
+    },
+    type: 'owning'
+  };
+
+  const stream = new ReadableStream(source);
+
+  const chunk = await stream.getReader().read();
+
+  assert_not_equals(chunk.value, port1);
+
+  let promise = new Promise(resolve => port2.onmessage = e => resolve(e.data));
+  chunk.value.postMessage("toPort2");
+  assert_equals(await promise, "toPort2");
+
+  promise = new Promise(resolve => chunk.value.onmessage = e => resolve(e.data));
+  port2.postMessage("toPort1");
+  assert_equals(await promise, "toPort1");
+}, 'Transferred MessageChannel works as expected');
+
+promise_test(async () => {
+  const channel = new MessageChannel;
+  const port1 = channel.port1;
+  const port2 = channel.port2;
+
+  const source = {
+    start(controller) {
+      controller.enqueue({ port1 }, { transfer : [ port1 ] });
+    },
+    type: 'owning'
+  };
+
+  const stream = new ReadableStream(source);
+  const [clone1, clone2] = stream.tee();
+
+  await clone2.getReader().read().then((value) => {
+    assert_unreached('clone2 should error');
+  }, () => { });
```

Comment:
Use one of the `promise_rejects_*` helpers instead:
```suggestion
  await promise_rejects_dom(t, "DataCloneError", clone2.getReader().read());
```
The `t` variable comes from the test function's parameter:
```js
promise_test(async t => {
```
(You may want to change this in all your new tests.)

Reply:
Done